### ISSUE TITLE
Make it possible to add headers to the Vegeta request

### ISSUE THREAD

**Issue by AndreMiras** [paradigmxyz/flood#29]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to test nodes that sit behind a proxy requiring a set of header, e.g. node providers that require an API key in the header.

**Describe the solution you'd like**
Ideally I'd like to be able to pass headers as Vegeta allows it (e.g. `-header` flag).
It could be via a command line flag or environment variable.

**Describe alternatives you've considered**
I hardcoded my header in the code to make it work for now:
https://github.com/paradigmxyz/flood/blob/0.2.5/flood/tests/load_tests/vegeta.py#L53

**Comment by sslivkoff** [comment]
fixed in 7cd1ac07c16c0a18bc191dc9746797e27809cb00

syntax = `--vegeta-args VEGETA_ARGS`

e.g. `--vegeta-args "-timeout 5s -cpus 1"`

**Comment by AndreMiras** [comment]
Pretty cool thanks!
I could indeed get things like `--vegeta-args "-cpus 1"` working, but `-header` seems to still be an issue, probably due to the way the quoted value passed after `-header` is being parsed e.g. `-header "Cache-Control: no-cache"`
Vegeta then complains with `invalid value "\"Cache-Control:" for flag -header: header '"Cache-Control:' has a wrong format`
Complete command used:
```
docker run --rm paradigmxyz/flood:latest eth_getBlockByNumber \
    node1=https://eth.llamarpc.com \
    node2=https://eth.llamarpc.com \
    --duration 3 --rate 1 --vegeta-args '-header "Cache-Control: no-cache"'
```
Error with more context:
```
┌─────────────────────────────────┐
│ Load test: eth_getBlockByNumber │
└─────────────────────────────────┘
- sample rates: [1]
- sample duration: 3
- extra args: -header "Cache-Control: no-cache"
- output directory: /tmp/tmps32wrbgm   

Gathering node data...
──────────────────────
[WARNING] ctc config file does not exist; use `ctc setup` on command line to generate a config file
[WARNING] ctc config file does not exist; use `ctc setup` on command line to generate a config file

       node  │                       url  │               metadata
    ─────────┼────────────────────────────┼─────────────────────────
      node1  │  https://eth.llamarpc.com  │  llamanodes_web3_proxy
             │                            │                v0.36.0     
    ─────────┼────────────────────────────┼─────────────────────────
      node2  │  https://eth.llamarpc.com  │  llamanodes_web3_proxy
             │                            │                v0.36.0  


Running load tests...
─────────────────────
[2023-07-23 12:09:11] Starting
[2023-07-23 12:09:11] Running load test for node1
[2023-07-23 12:09:12] Running attack at rate = 1 rps
invalid value "\"Cache-Control:" for flag -header: header '"Cache-Control:' has a wrong format
Usage of vegeta attack:
  -body string
        Requests body file
... truncating ...
  -workers uint
        Initial number of workers (default 10)
2
```
